ruby-test-mode.el

```diff
@@ -69,49 +70,92 @@ Test Driven Development in Ruby."
   "The keymap used in `ruby-test-mode' buffers.")
 
 (defcustom ruby-test-file-name-extensions
-  '("builder" "erb" "haml" "rb" "rjs")
+  '("builder" "erb" "haml" "rb" "rjs" "rake" "slim")
```

Comment:
> Is it common to have tests for those extensions?

I write view test very few,  though, rspec support write spec with view.
so, I think erb/haml/slim is equivalent.

here is one snippets code from my one project

```sh
╰─ $ cat index.html.slim_spec.rb 
require 'rails_helper'

RSpec.describe "complaints/index", type: :view do
  before(:each) do
    assign(:complaints, [
      Complaint.create!(
        :order => nil,
        :user => nil,
        :content => "MyText"
      ),
      Complaint.create!(
        :order => nil,
        :user => nil,
        :content => "MyText"
      )
    ])
  end

  it "renders a list of complaints" do
    render
    assert_select "tr>td", :text => nil.to_s, :count => 2
    assert_select "tr>td", :text => nil.to_s, :count => 2
    assert_select "tr>td", :text => "MyText".to_s, :count => 2
  end
```
rake task test is exist in my one project.

in fact, js test is common use in some project, so i think we could add
similiar mapper in ruby-test-mode 
the key is, I don't know the common spec which where to found js 
in ruby rails project.in my one project, exist following mapping.
app/assets/javascripts/application/aaa/bbb.coffee => spec/javascripts/aaa/bbb_spec.coffee

But, i don't know those if common enough to as a spec.

If performance is not a problem for this, we could add more
mapping/candidate, right ?